target/linux/ath79/dts/ar9342_ubnt_xw.dtsi

```diff
@@ -110,3 +117,15 @@
 	mtd-cal-data = <&eeprom 0x1000>;
 	mtd-mac-address = <&eeprom 0x1002>;
 };
+
+&eth0 {
+	mtd-mac-address = <&eeprom 0x0>;
+};
+
+&mdio1 {
```

Comment:
mdio1 is a subdevice of eth1 so it isn't enabled if eth1 is disabled. This isn't needed.